### ISSUE TITLE
MissingFieldException: 'known_for_department'

### ISSUE THREAD

**Issue by edmundjohnson** [MoviebaseApp/tmdb-api#9]
I'm getting an exception with v0.6.3 when calling
`Tmdb3.find.find("tt11271038", "en", TmdbExternalSource.IMDB) `

Stacktrace:

```
    kotlinx.serialization.MissingFieldException: 
     Field 'known_for_department' is required for type with serial name 'app.moviebase.tmdb.model.TmdbCrew', but it was missing
        at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
        at app.moviebase.tmdb.model.TmdbCrew.<init>(TmdbPeopleModel.kt:115)
        at app.moviebase.tmdb.model.TmdbCrew$$serializer.deserialize(TmdbPeopleModel.kt:115)
        at app.moviebase.tmdb.model.TmdbCrew$$serializer.deserialize(TmdbPeopleModel.kt:115)

```

**Comment by chrisnkrueger** [comment]
Thanks. I will check this field.

**Comment by chrisnkrueger** [comment]
The issue is fixed in version 0.6.4

**Comment by edmundjohnson** [comment]
Thank you!
